source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -47,7 +47,8 @@
 CONSISTENCY_HORIZON_DELTA: timedelta = timedelta(minutes=5)
 
 
-properties_cache: dict[str, Properties]= {}
+properties_cache: dict[str, Properties] = {}
```

Comment:
this file seems to be cosmetic and unrelated changes that can be backed out, although the types of `page` and `cutoff` are also changed (and shouldn't be?)